app/ships_neighboring.py

```diff
@@ -0,0 +1,18 @@
+from typing import List
+
+
+def are_ships_separated(ships: List[tuple]) -> None:
+    def are_cells_neighbors(cell1: tuple, cell2: tuple) -> bool:

```

Comment:
```suggestion
    def are_cells_neighbors(cell1: tuple[int], cell2: tuple[int]) -> bool:

```

review all annotations, please, and fix them all :)

Reply:
In module 'number_of_ships.py' -> def check_number_of_ships(fleet: list[Ship]) -> None:

In module 'ships_neoghboring.py' -> def are_cells_neighbors(cell1: tuple[int, int], cell2: tuple[int, int]) -> bool:

In module 'bounding_ships_and_decks.py' -> def create_field(fleet: list[Ship]) -> dict:

In module 'main.py' ->     def fire(self, location: tuple[int, int]) -> str: